packages/appeals-service-api/src/services/user.service.js

```diff
@@ -36,23 +32,22 @@ const getUsers = async (lpaCode) => {
 
 	try {
 		const cursor = await mongodb.get().collection('user').find({ lpaCode: lpaCode, enabled: true });
-		// to sort from mongo instead of locally
-		// .sort({ isAdmin: -1, email: 1 });
 
 		await cursor.forEach((doc) => {
 			result.push({
 				_id: doc._id,
 				email: doc.email,
 				lpaCode: doc.lpaCode,
-				isAdmin: doc.isAdmin
+				isAdmin: doc.isAdmin,
+				confirmedAt: doc.confirmedAt
 			});
 		});
 	} catch (err) {
 		logger.error(err);
 		throw err;
 	}
 
-	result.sort(compareUser);
+	result.sort(compareUserConfirmedAt);
```

Comment:
We could maybe do with a unit test on the date ordering part if we don't have one already.

Reply:
have unit tested the user service, although had a nightmare attempting to mock the lpaservice dependency so have left create user without some tests for now